less/patternfly-react.less

```diff
@@ -1,3 +1,31 @@
 /**
   Patternfly React Specific Extensions
 */
+
+.wizard-pf-header {
+  background-color: #f5f5f5;
+  border-bottom: none;
+  padding: 10px 18px;
+}
+.wizard-pf-title {
+  font-size: 13px;
+  font-weight: 700;
+}
+.wizard-pf-footer {
+  padding: 14px 15px 17px;
+  text-align: right;
+  & .btn + .btn {
+    margin-left: 5px;
+    margin-bottom: 0;
+  }
+  & > .btn {
+    padding-left: 10px;
+    padding-right: 10px;
+    & > .fa-angle-left {
+      margin-right: 5px;
+    }
+    & > .fa-angle-right {
+      margin-left: 5px;
+    }
+  }
+}
```

Comment:
I think it would be easier to maintain if you separate it into a different file `wizard.less`

Reply:
I agree completely... I think after reviewing it again, it is OK to remove that temporary Less for now (as plan is to push it to Core). Will update it...